### PR TITLE
Java: Support cs_strerror() and cs_regs_access()

### PR DESCRIPTION
**Pull request by akirschbaum**
Add two more functions to the Java binding: cs_strerror() as Capstone.strerror() and cs_regs_access() as CsInsn.regsAccess().

### CODE REVIEW

**Comment by akirschbaum** [comment]
Renamed the confused class name to CsRegsAccess which matches the name of the C function. Also rebased to 'next' branch.

File: bindings/java/capstone/Capstone.java [hunk 1]
```diff
@@ -238,6 +240,38 @@ public boolean group(int gid) {
       return cs.cs_insn_group(csh, raw.getPointer(), gid) != 0;
     }
 
+    public CsRegsAccess regsAccess() {
+      Memory regsReadMemory = new Memory(64*2);
```
Comment (aquynh) [inline bindings/java/capstone/Capstone.java]: could you explain why this is `64*2` here?

**Comment by aquynh** [comment]
merged, thanks!

**Comment by aquynh** [comment]
btw, we need a Java developer to review this pull req for Java binding at https://github.com/aquynh/capstone/pull/609. would you please take a look at that?

**Pull request merged**